workflow/scores.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import division
+
+import itertools
+import numpy as np
+
+from iou import cc_iou as iou
+
+
+def score_craters_on_patch(y_true, y_pred):
+    """
+    Main score
+
+    Parameters
+    ----------
+    y_true : list of tuples (x, y, radius)
+        List of coordinates and radius of actual craters in a patch
+    y_pred : list of tuples (x, y, radius)
+        List of coordinates and radius of craters predicted in the patch
+
+    Returns
+    -------
+    float : score for a given path, the higher the better
+
+    """
+    y_true = np.atleast_2d(y_true).T
+    y_pred = np.atleast_2d(y_pred).T
+
+    ospa_score = ospa(y_true, y_pred)
+
+    score = 1 - ospa_score
+
+    return score
+
+
+def score_iou(y_true, y_pred):
+    pass
+
+
+def score_completeness(y_true, y_pred):
+    pass
+
+
+def ospa(x_arr, y_arr, p_norm=1, cut_off=1):
+    """
+    Optimal Subpattern Assignment (OSPA) metric for IoU score
+
+    This metric provides a coherent way to compute the miss-distance
+    between the detection and alignment of objects.
```

Comment:
maybe explain a bit more the rationale? 
something like "finds the best aligment of the true/predicted objects (alignment that gives the lowest sum of distances)"